src/main/java/com/blibli/future/vo/ApprovalRequestVo.java

```diff
@@ -23,7 +25,7 @@ public void setNik(String nik) {
 		this.nik = nik;
 	}
 
-	public boolean isApproved() {
+	public boolean getIsApproved() {
```

Comment:
https://stackoverflow.com/questions/21913955/json-post-request-for-boolean-field-sends-false-by-default